app/src/main/java/org/systers/mentorship/models/HomeStatistics.kt

```diff
@@ -0,0 +1,36 @@
+package org.systers.mentorship.models
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * This class represents statistics of the user actions on the app.
+ * @param name the name of the user
+ * @param pendingRequests number of pending requests
+ * @param acceptedRequests number of accepted requests
+ * @param completedRelations number of completed relations
+ * @param cancelledRelations number of cancelled relations
+ * @param rejectedRequests number of rejected requests
+ * @param achievements a list of upto 3 completed tasks
+ */
+
+@SuppressLint("ParcelCreator")
```

Comment:
Remove this. Update your IDE. This bug has been fixed

Reply:
This was not a problem with my IDE, but with Travis. The lint check on Travis failed without this annotation. This is the error that is thrown during Travis' lint check:
`This class implements Parcelable but does not provide a CREATOR field [ParcelCreator]`